go.mod

```diff
@@ -70,9 +70,12 @@ require (
 	github.com/gosimple/slug v1.14.0
 	github.com/influxdata/influxdb1-client v0.0.0-20220302092344-a9ab5670611c
 	github.com/jackc/pgx/v4 v4.18.3
+	github.com/jackc/pgx/v5 v5.6.0
+	github.com/jaswdr/faker v1.19.1
```

Comment:
We already include `github.com/go-faker/faker/v4`. Would be best to use that one instead.